system_test/helpers/aest_api.erl

```diff
@@ -45,57 +38,33 @@
 
 %=== API FUNCTIONS =============================================================
 
--spec get_balance(atom(), account(), aest_nodes:test_ctx()) -> non_neg_integer().
-get_balance(NodeName, Account, Cfg) ->
-    api_get_balance(NodeName, Account, Cfg).
-
--spec tx_spend(atom(), account(), account(), pos_integer(), pos_integer(),
-               aest_nodes:test_ctx())
-    -> binary().
-tx_spend(NodeName, From, To, Amount, Fee, Cfg) ->
-    #{ pubkey := FromPubKey } = From,
-    #{ pubkey := ToPubKey } = To,
-    TxArgs = #{
-        sender_id        => FromPubKey,
-        recipient_id     => ToPubKey,
-        amount           => Amount,
-        fee              => Fee,
-        payload          => <<"foo">>
-    },
-    {TxHash, Tx} = tx_prepare(NodeName, spend_tx, From, TxArgs, Cfg),
-    tx_post(NodeName, TxHash, Tx, Cfg).
-
--spec tx_state(atom(), binary(), aest_nodes:test_ctx()) -> tx_state().
-tx_state(NodeName, TxHash, Cfg) ->
-    case api_get_tx(NodeName, TxHash, Cfg) of
-        {ok, 404, _} ->
-            undefined;
-        {ok, 200, #{ <<"transaction">> := #{ <<"block_hash">> := <<"none">> }} } ->
-            mempool;
-        {ok, 200, #{ <<"transaction">> := #{ <<"block_hash">> := _Other }} } ->
-            chain
-    end.
-
--spec tx_wait(atom() | [atom()], binary(), tx_state(), pos_integer(),
-              aest_nodes:test_ctx())
-    -> ok | timeout.
-tx_wait(NodeNames, TxHash, Status, Timeout, Cfg) ->
-    CheckFun = fun(NodeName) -> tx_state(NodeName, TxHash, Cfg) =:= Status end,
-    wait_for(NodeNames, CheckFun, 100, Timeout).
-
 -spec sc_open(sc_open_params(), aest_nodes:test_ctx())
     -> {ok, channel(), binary(), pos_integer()}.
 sc_open(Params, Cfg) ->
     #{
-        initiator_node := INodeName,
-        initiator_id   := IAccount,
-        responder_node := RNodeName,
-        responder_id   := RAccount
+        initiator_node    := INodeName,
+        initiator_id      := IAccount,
+        initiator_amount  := IAmt,
+        responder_node    := RNodeName,
+        responder_id      := RAccount,
+        responder_amount  := RAmt
     } = Params,
     #{ pubkey := IPubKey, privkey := IPrivKey } = IAccount,
     #{ pubkey := RPubKey, privkey := RPrivKey } = RAccount,
 
-    Opts = sc_options(Params, #{}, Cfg),
+    {Host, _Port} = node_ws_int_addr(RNodeName, Cfg),
```

Comment:
I see this calls `aest_nodes:get_internal_address(NodeName, ext_ws, Cfg)` - I need to understand how it works `get_internal_address` but with `ext_ws` as arg. (It seems making `int_ws` unused, though maybe out of scope for this PR.)

Reply:
I see int_ws would be for chain WS API.

Reply:
I see this node_ws_int_addr was factored from other portion of the code so out of scope for this PR.